nan_json.h

```diff
@@ -66,18 +67,20 @@ class JSON {
   static v8::Local<v8::Value> Call(const char *method,
     int argc, v8::Local<v8::Value> *argv) {
     v8::Local<v8::Value> globalJSON =
-      Nan::GetCurrentContext()->Global()->Get(
+      Nan::Get(
+        Nan::GetCurrentContext()->Global(),
         Nan::New("JSON").ToLocalChecked()
-      );
+      ).ToLocalChecked();
 
     if (!globalJSON->IsObject()) {
       return Nan::Undefined();
     }
 
-    v8::Local<v8::Object> json = globalJSON->ToObject();
+    v8::Local<v8::Object> json =
+      Nan::To<v8::Object>(globalJSON).ToLocalChecked();
 
     v8::Local<v8::Value> thisMethod =
-      json->Get(Nan::New(method).ToLocalChecked());
+      Nan::Get(json, Nan::New(method).ToLocalChecked()).ToLocalChecked();
 
     if (thisMethod.IsEmpty() || !thisMethod->IsFunction()) {
```

Comment:
fixed... I updated all three branches to always check `MaybeLocal.IsEmpty()` before proceeding to `ToLocalChecked()`     ...  is this the right way to do it?  Unfortunately, I haven't been able to find much in the _NAN_ documentation about how to check a `MaybeLocal` but I found `IsEmpty()` in the _V8_ documentation.